### PR TITLE
[WIP] redesign agency people home view #563 #554 #565

### PR DESCRIPTION
**Pull request by Thomascountz**
**I need help refactoring this.**
Should we create partials? I'm having trouble because of the need for `active` classes for the `nav-tab`s and `tab-pane`s, depending on the user's role.

I'm also having trouble decreasing the length of many lines.

Known issues:
- Hound: many lines too long
- View for user who is both a case manager and job developer, the bootstrap pills are wrapping (see img)
- Code is not DRY (repeated CSS `id`s)

screen shot 2017-03-21 at 4 38 57 pm

### CODE REVIEW

**Comment by patmbolger** [comment]
@Thomascountz - I made some changes to the AP home page - see my test PR: https://github.com/AgileVentures/MetPlus_PETS/pull/545

File: app/views/agency_people/home.html.haml [hunk 1]
```diff
@@ -1,70 +1,148 @@
...
-      - if @agency_person.is_agency_admin?(@agency)
-        %li(role="presentation")
-          %a(href="#tab3" aria-controls="tab3" role="tab" data-toggle="tab")
-            All Agency Open Tasks
-      %li(role="presentation")
-        %a(href="#tab4" aria-controls="tab4" role="tab" data-toggle="tab")
+.row
+  .col-sm-12
+    %h2
+      = "Welcome back to PETS, #{pets_user.full_name(last_name_first: false)}"
```
Comment (Thomascountz) [inline app/views/agency_people/home.html.haml]: We need a double quoted string to allow for interpolation.

**Comment by Thomascountz** [comment]
reopen to run tests

**Comment by Thomascountz** [comment]
reopen to run tests

**Pull request merged**